core/src/main/java/hudson/ClassicPluginStrategy.java

```diff
@@ -206,17 +207,11 @@ private static Manifest loadLinkedManifest(File archive) throws IOException {
             if (libs != null)
                 paths.addAll(Arrays.asList(libs));
 
-            try {
-                Class pathJDK7 = Class.forName("java.nio.file.Path");
-                Object toPath = File.class.getMethod("toPath").invoke(expandDir);
-                URI uri = (URI) pathJDK7.getMethod("toUri").invoke(toPath);
+            Path toPath = expandDir.toPath();
```

Comment:
Just inlined those local variables btw. About your question, well, I don't even see the @since in the code.